README.md

```diff
@@ -18,6 +18,12 @@ For all other storage systems, please check with your storage vendor.  If your s
 durable snapshot storage you may want to consider 
 [Velero's Restic Integration](https://velero.io/docs/latest/restic)
 
+## Verified environment
+Velero CSI plugin v0.3 is tested with Velero v1.9 on the following environments:
+* EKS: ebs.csi.aws.com driver is tested with helm chart version helm-chart-aws-ebs-csi-driver-2.6.8 and external-snapshotter version is v5.0.1. v6.0.1 external-snapshotter is also tested, but aws-ebs-csi-driver doesn't have a official helm chart tag supporting v6.0.1 external-snapshotter yet.
```

Comment:
There is a tag for aws-ebs-driver that pins to v6.0.1:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases/tag/helm-chart-aws-ebs-csi-driver-2.6.11

And I think this is what we used after v0.3.0-rc1 to verify the functionalities.

Reply:
Got it. Thanks.